script.js

```diff
@@ -83,61 +88,235 @@ const data = [
+const liveImg = document.createElement('img');
+liveImg.setAttribute('src', 'assets/image/livecon.png');
+modalLive.appendChild(textModalLive);
+modalLive.appendChild(liveImg);
+modalLiveDiv.appendChild(modalLive);
+divSrcLink.appendChild(modalLiveDiv);
+// source code
+const srcode = document.createElement('div');
+srcode.setAttribute('class', 'madbtnx');
+const srclink = document.createElement('a');
+srclink.setAttribute('class', 'srclink');
+srclink.setAttribute('href', 'https://github.com/aneallaryea100/Portfolio');
+const srctxt = document.createTextNode('See Source');
+const srcImg = document.createElement('img');
+srcImg.setAttribute('src', 'assets/image/gitsrc.png');
+srclink.appendChild(srctxt);
+srcode.appendChild(srclink);
+srcode.appendChild(srcImg);
+divSrcLink.appendChild(srcode);
+
+// midal functions
+// Get the modal
+const errst = document.querySelectorAll('.erest');
+// Get the <span> element that closes the modal
+const span = document.getElementsByClassName('spanz')[0];
+// When the user clicks the button, open the modal
+errst.forEach((i) => {
+  i.onclick = function () {
+    console.log('call');
+    midal.style.display = 'block';
+  };
+});
+// When the user clicks on <span> (x), close the modal
+span.onclick = function () {
+  midal.style.display = 'none';
+};
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function (event) {
+  if (event.target === midal) {
+    midal.style.display = 'none';
+  }
+};
+
+/** *****************************separate cardss******************************* */
+const wert = document.getElementById('cardSecs');
+
+const createGridCards = (objTest) => {
+  const divHse = document.createElement('div');
+  divHse.setAttribute('class', 'divhse');
+  const divhhd = document.createElement('div');
+  divHse.appendChild(divhhd);
+  divhhd.setAttribute('class', 'hhdiv');
+  const ddelhed = document.createElement('h2');
+  divHse.appendChild(ddelhed);
+  ddelhed.setAttribute('class', 'head2');
+  ddelhed.innerHTML = objTest.name;
+  // text
+  const ttlorem = document.createElement('p');
+  ttlorem.setAttribute('class', 'rtex');
+  ttlorem.innerHTML = objTest.description;
+  divHse.appendChild(ttlorem);
+  // list techs
+  const hslist = document.createElement('ul');
+  divHse.appendChild(hslist);
+  hslist.setAttribute('class', 'listHse');
+  const llul = document.createElement('li');
+  llul.setAttribute('class', 'tyres');
+  llul.innerHTML = 'HTML';
+  hslist.appendChild(llul);
+  const llul2 = document.createElement('li');
+  hslist.appendChild(llul2);
+  llul2.setAttribute('class', 'tyres');
+  llul2.innerHTML = 'bootstrap';
+  const llul3 = document.createElement('li');
+  hslist.appendChild(llul3);
+  llul3.setAttribute('class', 'tyres');
+  llul3.innerHTML = 'Ruby';
+  // button
+  const btxx = document.createElement('button');
+  btxx.setAttribute('class', 'erest');
+  btxx.innerHTML = 'See Projects';
+  divHse.appendChild(btxx);
+
+  return divHse;
+};
```

Comment:
* Popup now meets Figma's requirement.
* Background is blurred just like on Figma.
* In this project, the minimum requirement is to meet the mobile and desktop views on Figma. 
* The overlapping issue requirement means I will have to introduce a new media query to meet the requirement to prevent any overlapping.